backend/src/controllers/formContoller.ts

```diff
@@ -41,3 +55,63 @@ export const getForms = async (req: any, res: Response) => {
     res.status(400).json({ error });
   }
 };
+
+export const updateForm = async (req: Request, res: Response) => {
+  try {
+    const { formId } = req.params;
+    const validationResult: Joi.ValidationResult<CreateFormType> =
+      editFormValidation.validate(req.body);
+    if (validationResult.error) {
+      return res.status(400).json({ message: validationResult.error.message });
+    }
+
+    const { title, description }: CreateFormType = req.body;
```

Comment:
Same here all these logic should reside in the service layer, the controller should only validate the input data and fwd to the service the business logic part